src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,83 @@
 import React from 'react';
+import cn from 'classnames';
+import PropTypes from 'prop-types';
 import './TodoList.scss';
 
-export const TodoList = () => (
+export const TodoList = ({
+  todos,
+  onUserSelect,
+  onChangeSearchField,
+  onChangeSelect,
+}) => (
   <div className="TodoList">
     <h2>Todos:</h2>
 
     <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
+      <div className="TodoList__form">
+        <input
+          className="TodoList__search"
+          type="text"
+          onChange={(event) => {
+            onChangeSearchField(event.target.value);
+          }}
+        />
+        <select
+          className="TodoList__search"
+          name="completed"
+          onChange={(event) => {
+            onChangeSelect(event.target.value);
+          }}
```

Comment:
these functions could be combined into one, just use target.name to determine which element is updated